misc/py_wrap_generator.py

```diff
@@ -630,6 +631,14 @@ def from_string(str_def, containing_file, line_number):
 			else:
 				arg.wtype.attr_type = attr_types.amp
 				arg.varname = arg.varname[1:]
+
+		# special exception: format strings
+		if arg.wtype.name in ["std::string_view", "string_view"]:
+			if arg.varname == "format":
+				arg.coerce_arg = '"%s"'
+			elif arg.varname == "prefix" and "warning" in owner_fn_name:
+				arg.coerce_arg = '"Warning: "'
```

Comment:
```suggestion
				arg.coerce_arg = '"Warning: "'
	        else:
	            arg.wtype.name = "std::string"
```

`string_view` isn't supported by boost::python so this will convert the argument for functions like `log_file_info` that take a `string_view` (at the cost of a copy).